pkgs/development/python-modules/pyscf/default.nix

```
@@ -1,57 +1,143 @@
 { lib
 , buildPythonPackage
+, isPy27
 , fetchFromGitHub
 , cmake
 , openblas
 , libcint
 , libxc
+, xcfun
 , h5py
 , numpy
 , scipy
-, pytest
+  # Check Inputs
+, nose
+, nose-exclude
 }:
 
 buildPythonPackage rec {
   pname = "pyscf";
-  version = "1.7.0";
+  version = "1.6.6";
 
   # must download from GitHub to get the Cmake & C source files
   src = fetchFromGitHub {
     owner = "pyscf";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0gdmc04nr699bq6gkw9isfzahj0k2gqhxnjg6gj9rybmglarkn15";
+    sha256 = "097cm1823r1v3hv7nk3m2k2xg2z4knsczhahxdgipwgrpank8fab";
   };
 
+  disabled = isPy27;
+
   nativeBuildInputs = [ cmake ];
 
   buildInputs = [
     libcint
     libxc
     openblas
+    xcfun
   ];
+
+  prePatch = ''
+    mkdir -p ./pyscf/lib/deps/include ./pyscf/lib/deps/lib
+    ln -s ${lib.strings.makeSearchPath "include" [xcfun] }/xc.h ./pyscf/lib/deps/include/xcfun.h
+    ln -s ${lib.strings.makeLibraryPath [ xcfun ] }/libxc.so ./pyscf/lib/deps/lib/libxcfun.so
+  '';
+
+
   cmakeFlags = [
     # disable rebuilding/downloading the required libraries
     "-DBUILD_LIBCINT=0"
     "-DBUILD_LIBXC=0"
     "-DBUILD_XCFUN=0"
+    "-DENABLE_XCFUN=1"
   ];
-  dontUseCmakeConfigure = true;
+  # Configure CMake to build C files in pyscf/lib
+  preConfigure = ''
+    pushd pyscf/lib
+  '';
```

might not work depending on PWD
```suggestion
  cmakeDir = "pyscf/lib";
```

---

I tried this, but the rest of the python build process expects files in ``PWD/pyscf/lib/build``.